cdap-distributions/src/hdinsight/pageblob-configure.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+#
+# Copyright © 2016 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+#
+# Configure HDFS Pageblob support for CDAP HDInsight cluster
+#
+
+# Ambari constants
+AMBARICONFIGS_SH=/var/lib/ambari-server/resources/scripts/configs.sh
+AMBARIPORT=8080
+ACTIVEAMBARIHOST=headnodehost
+
+# Function definitions
+
+die() { echo "ERROR: ${*}"; exit 1; };
```

Comment:
Since this script uses multiple exit codes, you should set one up...
```
die() { __code=${2:-1}; echo "ERROR: ${1}"; exit ${__code}; };
```
The only disadvantage to this is then you'll be forced to put everything in quotes to make it a single parameter.